modules/common.nix

```diff
@@ -20,6 +20,10 @@ in {
   # Non-root users are not allowed to install authorized keys.
   services.openssh.authorizedKeysFiles = pkgs.lib.mkForce
     [ "/etc/ssh/authorized_keys.d/%u" ];
+  services.openssh.extraConfig = lib.mkOrder 999 ''
```

Comment:
@cleverca22 can't we just put `/root/.ssh/authorized_keys` into services.openssh.authorizedKeysFiles?

Reply:
@domenkozar, that was the first question I asked Michael yesterday -- and he suggested that the option is effective for _every_ user, which is somewhat conceptually ugly.